samples/Mvc.LocalizationSample.Web/Models/Product.cs

```diff
@@ -7,10 +7,10 @@ namespace Mvc.LocalizationSample.Web.Models
 {
     public class Product
     {
-        [Required(ErrorMessage = "ProductName")]
+        [Required(ErrorMessage = "Product Name is invalid")]
```

Comment:
`"{0} is invalid."`

Use `[Display(Name = "Product Name")]` to change the name shown. Add resources for `"Product Name"` if that should also be localized. Similar comments in other places where you're not displaying the property name.